studio/components/head/Favicons.tsx

```diff
@@ -22,15 +22,15 @@ const Favicons = () => {
       {/* prettier-ignore */}
       <link rel="apple-touch-icon-precomposed" sizes="152x152" href={`${basePath}/favicon/apple-touch-icon-152x152.png`}/>
       {/* prettier-ignore */}
-      <link rel="icon" type="image/png" href={`${basePath}/favicon/favicon-196x196.png" sizes="196x196`}/>
```

Comment:
same again

Reply:
The quotes were off actually, fixed it 👍